library/init/meta/interactive.lean

```diff
@@ -6,8 +6,11 @@ Authors: Leonardo de Moura
 prelude
 import init.meta.tactic init.meta.rewrite_tactic init.meta.simp_tactic
 import init.meta.smt.congruence_closure init.category.combinators
+import init.meta.lean_parser
 
 namespace interactive
+@[reducible] meta def parse {α : Type} (p : lean_parser α) : Type := α
```

Comment:
Ah, I see.  The `itactic` type was unfortunately missing in Sebastian's mockup.  I see two options:
 * Completely weakly typed.
 * With the `has_quote` approach, we just need a type of pre-expressions that does not get quoted (that is, it has the right has_quote instance).  I suggested `meta def antiquote_pexpr := pexpr`.